clang/test/Driver/sycl-offload.c

```diff
@@ -671,3 +671,11 @@
 // OFFLOAD-NEW-DRIVER: 18: backend, {17}, assembler, (host-sycl)
 // OFFLOAD-NEW-DRIVER: 19: assembler, {18}, object, (host-sycl)
 // OFFLOAD-NEW-DRIVER: 20: clang-linker-wrapper, {19}, image, (host-sycl)
+
+/// Check the toolflow for SYCL compilation using new offload model
+// RUN: %clangxx -### -fsycl --offload-new-driver %s 2>&1 | FileCheck -check-prefix=CHK-FLOW %s
```

Comment:
Please, specify target and -fsycl-targets. They are checked below and these checks will pass only in specific environments.